sdcm/nemesis.py

```diff
@@ -344,7 +344,7 @@ def upgrade_node(self, node):
         node.remoter.run('sudo chown root.root /etc/yum.repos.d/scylla.repo')
         node.remoter.run('sudo chmod 644 /etc/yum.repos.d/scylla.repo')
         node.remoter.run('sudo yum clean all')
-        node.remoter.run('sudo yum update scylla scylla-conf scylla-server scylla-jmx scylla-tools -y')
+        node.remoter.run('sudo yum update scylla -y')
```

Comment:
I'm not sure upgrading all packages is the best idea here. Are you having any troubles with this command upgrading only the scylla packages?